core/client/fs/src/main/java/alluxio/client/file/FileSystemContext.java

```diff
@@ -141,14 +147,14 @@ public void close() throws IOException {
     mFileSystemMasterClientPool = null;
     mBlockMasterClientPool.close();
     mBlockMasterClientPool = null;
+    mMasterInquireClient = null;
```

Comment:
Should we null the netty pools?

Reply:
closing and clearing them below should be good enough - the empty map can be re-used safely